src/Components/Sidebar.jsx

```diff
@@ -0,0 +1,230 @@
+import {
+  Flex,
+  Drawer,
+  Box,
+  DrawerBody,
+  Tooltip,
+} from "@sparrowengg/twigs-react";
+
+import MenuIcon from "./../asset/menu.svg";
+import SideBarIcon1 from "./../asset/sidebarImg1.svg";
+import SideBarIcon2 from "./../asset/sidebarImg2.svg";
+import SideBarIcon3 from "./../asset/sidebarImg3.svg";
```

Comment:
@9xVibee can you use one import file which will take all these imports and try to use that here